### PR TITLE
C3DC-892 Announcements in Navbar

### PR DESCRIPTION
**Pull request by huuaho**
[C3DC-892](https://tracker.nci.nih.gov/browse/C3DC-892)

The announcement page has been added as an option under the about dropdown in the nav bar.
This change involved a minor update to the URLs of the latest release and user guide as well.
Some minor linting errors have been resolved as well.

### CODE REVIEW

File: src/bento/globalHeaderData.js
Comment (Senait7-ux) [inline src/bento/globalHeaderData.js:1]: The announcement section looks good , however I am seeing unnecessary updates regarding the pdfReader which is not part of this PR. Is there any valid reason for changing the  links for the latest release and the user guid? if so  comment it on the PR.

File: src/components/Layout/LayoutView.js
Comment (Senait7-ux) [inline src/components/Layout/LayoutView.js:1]: can you remove line 40? the commented route

File: src/pages/pdfReader/pdfReader.js
Comment (Senait7-ux) [inline src/pages/pdfReader/pdfReader.js:1]: any valid reason for changing the pdf reader implementation from using a SearchParamter to a direct route? because a user might accidentally share the link as user_guid/ or release_note/ which will not render anything since we are using substring(1) , it will only work if the end url does not have '/',
eg. http://localhost:3000/user_guide/.   (will not work)
eg. http://localhost:3000/user_guide (will work)

Comment (huuaho) [reply src/pages/pdfReader/pdfReader.js:1]: Release Notes: Redirects to the Release Notes page “/release_notes” in the same window.
User Guide: Redirects to the User Guide page “/user_guide” in the same window.

^ In addition to the announcements page linkout in the navbar, the spec defined the urls to the user guide and release notes to have the above URLs

Comment (Senait7-ux) [reply src/pages/pdfReader/pdfReader.js:1]: if that is the case , let's just remove the trailing slash after the pathname has been declared,
if this helps 
 `// this will remove any trailing slashes that might be added by the end user accidentally 
pathname.replace("/","");
`
i will approve the PR right after this

**Review by Senait7-ux — APPROVED** [review]
LGTM

File: src/pages/pdfReader/pdfReader.js
Comment (Senait7-ux) [reply src/pages/pdfReader/pdfReader.js:1]: Resolved

File: src/components/Layout/LayoutView.js
Comment (Senait7-ux) [reply src/components/Layout/LayoutView.js:1]: Resolved
